mdpow/workflows/dihedrals.py

```diff
@@ -490,13 +502,20 @@ def dihedral_violins(df, width=0.9, solvents=SOLVENTS_DEFAULT, plot_title=None):
 
        *solvents*
            The default solvents are documented under :data:`SOLVENTS_DEFAULT`.
+
+       *plot_title*
+           format: f'{molname}, {name[0]} {a} | ''{col_name}'
+           determined by,
```

Comment:
Do you mean that the format string is determined by `build_svg()`? If so, make it part of the sentence and do not use seealso. If there's a hyperlink to follow as part of an explanatory sentence then that's good enough, no need for extra highlighting.

Reply:
seealso is typically use for broader context and related functionality where the connection may not be obvious from the other context